### ISSUE TITLE
Accessibility issue: b-form-datepicker

### ISSUE THREAD

**Issue by apavelm** [bootstrap-vue/bootstrap-vue#6597]
### Ensures buttons have discernible text

Accessibility check found violations on b-form-datepicker

- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty
- Element has no title attribute

declaration of the component

```
<b-form-datepicker id="someid"
                             today-button
                             close-button
                             locale="en"
                             v-model="somevar"
                             :start-weekday="'1'"
                             :date-format-options="{ year: 'numeric', month: 'long', day: '2-digit' }">
          </b-form-datepicker>

```
### Expected behavior

No WCAG violations

### Versions

**Libraries:**

- BootstrapVue: 2.21.2
- Bootstrap: 4.5.3
- Vue: 2.6.12
